deployment/event-generator.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: falco-event-generator-deployment
+  labels:
+    app: falco-event-generator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: falco-event-generator
+  template:
+    metadata:
+      labels:
+        app: falco-event-generator
+    spec:
+      containers:
+      - name: falco-event-generator
+        image: falcosecurity/event-generator
+        command: ["./event-generator", "run"]
```

Comment:
Atm, that's OK. When a `--loop` flag has been implemented I will add it here. I believe it makes sense just for the k8s deployment to generate events in a loop.